loader/src/main/java/net/neoforged/fml/loading/FMLConfig.java

```diff
@@ -23,10 +23,12 @@
 public class FMLConfig
 {
     public enum ConfigValue {
+        DISABLE_CONFIG_WATCHER("disableConfigWatcher", Boolean.FALSE, "Disables File Watcher. Used to automatically update config if its file has been modified."),
         EARLY_WINDOW_CONTROL("earlyWindowControl", Boolean.TRUE, "Should we control the window. Disabling this disables new GL features and can be bad for mods that rely on them."),
         MAX_THREADS("maxThreads", -1, "Max threads for early initialization parallelism,  -1 is based on processor count", FMLConfig::maxThreads),
         VERSION_CHECK("versionCheck", Boolean.TRUE, "Enable NeoForge global version checking"),
         DEFAULT_CONFIG_PATH("defaultConfigPath", "defaultconfigs", "Default config path for servers"),
+        CONFIG_BACKUPS_COUNT("configBackupsCount", 5, "Sets the number of backups for configs."),
```

Comment:
It was a TODO in the code, yet configuring a magic number like that feels kind of wrong?

Reply:
Maybe, but it's a good option (in my opinion).

Reply:
This does not really solve anything. The reason for the hardcode of the backups is that the fml config itself might fail to load and you then have no idea about the backup count the user is requesting.

I think for stability reasons this value should be hardcoded.